### PR TITLE
Add Post Haste 2.5.8.0

### PR DESCRIPTION
**Pull request by NaanProphet**
Created cask for Post Haste, an app for organizing projects with
customizable templates for post-production professionals.

### CODE REVIEW

File: Casks/post-haste.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  homepage 'https://www.digitalrebellion.com/posthaste/'
+  license :gratis
+
+  app 'Post Haste.app'
+
+  zap :delete => [
+                   '~/Library/Application Support/Digital Rebellion/Post Haste/',
+                   '~/Library/Preferences/com.digitalrebellion.PostHaste.plist',
+                 ]
+  zap :rmdir => '~/Library/Application Support/Digital Rebellion'
```
Comment (jawshooah) [inline Casks/post-haste.rb]: No need to use multiple `zap` stanzas, just add a comma after the closing `]` above.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 2bb5783ab9f0643f7a35f8d21a8dc71f127a1669. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by adidalal** [comment]
`bundle exec rubocop $cask_name --auto-correct` will fix errors for you FYI (ie, realigning zap commands after removing the extra `zap`)

**Comment by SirDap** [comment]
@jawshooah Ah thanks for the tip for combing the `zap` stanzas! For what it's worth, i did run the `brew cask style` before committing but apparently it passed.

**Comment by jawshooah** [comment]
Yeah, there's currently no check for duplicate stanzas in place.

**Comment by jawshooah** [comment]
Some stanzas, like `url` or `homepage`, will actually break the cask if they appear more than once. `zap` stanza contents can be merged, so having multiple won't actually break anything, it's just unnecessary.

**Comment by SirDap** [comment]
Ah i see. Right on!

**Pull request closed without merging**